hs_core/models.py

```diff
@@ -86,7 +87,11 @@ def can_add(self, request):
     def can_delete(self, request):
```

Comment:
@selimnairb Good point. Will address all your comments and post code update shortly.